flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorTest.java

```diff
@@ -124,9 +124,15 @@ public void testGetNonExistingFunction() throws Exception {
 	public void testDifferentFormats() throws Exception {
 		String[] formats = new String[]{"orc", "parquet", "sequencefile", "csv"};
 		for (String format : formats) {
-			if (format.equals("orc") && HiveShimLoader.getHiveVersion().startsWith("2.0")) {
-				// Ignore orc test for Hive version 2.0.x for now due to FLINK-13998
-				continue;
+			if (format.equals("orc")) {
+				String version = HiveShimLoader.getHiveVersion();
+				// hive 2.0.x with orc can not run in local mode, because Hadoop23Shims only
+				// supported for DFS, but can work in cluster.
+				// hive 2.2.x with orc can not run, because hive 2.2.x exec lack orc classes,
```

Comment:
Why could 2.2.x run ORC test before this PR? It doesn't feel right that we're adding shim and our test covers fewer versions.

Reply:
I'll add deps to 2.2.0 profile.